model/bitwarden/cipher.go

```diff
@@ -120,4 +122,28 @@ func FindCiphersInFolder(inst *instance.Instance, folderID string) ([]*Cipher, e
 	return ciphers, nil
 }
 
+// DeleteUnrecoverableCiphers will delete all the ciphers that are not shared
+// with the cozy organization. It should be called when the master password is
+// lost, as there are no ways to recover those encrypted ciphers.
+func DeleteUnrecoverableCiphers(inst *instance.Instance) error {
+	var ciphers []couchdb.Doc
+	err := couchdb.ForeachDocs(inst, consts.BitwardenCiphers, func(_ string, data json.RawMessage) error {
```

Comment:
Well, technically, we could, but it would be less efficient. We don't have an index for that, and even if we had an index, it would certainly need to be rebuilt. And, if we don't use an index, as most ciphers are not shared with the Cozy, I think it would be slower to perform the filtering in CouchDB that it is in go.

Reply:
Most people, even geeks, won't have more than one or two hundred of ciphers. Iterating is ok for me